### ISSUE TITLE
Support page doesn't work

### ISSUE THREAD

**Issue by m4hdi1995** [3s3s/opentrade#289]
Hello,
When I users try to submit a ticket on support page it doesn't work , it doesn't return any errors, anybody face that issue?

**Comment by freakcoderz** [comment]
Yes, but only because my email is not fully functional to send those emails out.

**Comment by m4hdi1995** [comment]
@freakcoderz but mine is , it's fully functional.

**Comment by freakcoderz** [comment]
I changed my support link to send people to Discord until I have a chance to mess with it. I did get a blank page before though.

**Comment by IZEM-Project** [comment]
Just edit support.ejs and insert any link for go to your email example <a href="mailto:user@gmail.com subject=subject text">Support Mail</a>

**Comment by m4hdi1995** [comment]
@IZEM-Project i thought it should be some ticketing system or something

**Comment by Xerxes7777** [comment]
I am using version 2 of the opentrade (the one that uses three main.js 
I tried test email  the opentrade website returns:
Success! Your message has been sent to support

but I don't receive any email address I set  up in the constants.js  in the server folder
Do i need to open a port?
I am using gmail.com email. 

What I am missing. 
Everything else seems to working this is the last hurdle. 
Thanks for your help

**Comment by freakcoderz** [comment]
You need to set up a way for your system to send that mail out. Usually this is broken on the VPS or never set up correctly.

**Comment by Xerxes7777** [comment]
> 
> 
The email feature works such as the temporary pin sent for each sign in. Also when you withdrawal fund from your wallet it email the temporary pin so these email are getting out of  VPS. 

But I can check the cloud sever ubuntu setting . I don't know how. Please let me know  thanks in advance

> You need to set up a way for your system to send that mail out. Usually this is broken on the VPS or never set up correctly.

**Comment by freakcoderz** [comment]
I never actually set it up. I had the link send them to my Discord server. I haven't used this in a few months as it is slow and clunky.

**Comment by Xerxes7777** [comment]
Ok I feel silly for some reason Gmail puts the tickets generated by the support page in to Junk or spam. I imagine this would not be the case with other email account. Also if you set up your account with Gmail it puts your set up verification information from the open trade site is sent to spam or junk mail. So have your user look there if they have problem setting their accounts up
